### PR TITLE
project: Minor save data fixes

### PR DESCRIPTION
**Pull request by hemangsk**

### CODE REVIEW

**Comment by hemangsk** [comment]
@surender7790 please have a look

File: models/candidates_2017.js [hunk 1]
```diff
@@ -2,18 +2,212 @@
 
 module.exports = function(sequelize, DataTypes) {
   return sequelize.define('candidates_2017', {
-    id: {
-      type: DataTypes.INTEGER(10).UNSIGNED.ZEROFILL,
-      allowNull: false,
-      primaryKey:true,
-      autoIncrement: true
-    },
+      AadharId: {
```
Comment (surender7790) [inline models/candidates_2017.js:5]: Why do we need all information in table 'candidates_2017'

File: models/applicant.js [hunk 2]
```diff
@@ -205,13 +201,14 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.STRING(10000),
       allowNull: true
     },
-    id: {
```
Comment (surender7790) [inline models/applicant.js:208]: ApplicantId ? why

File: config/verify.js [hunk 3]
```diff
@@ -39,9 +40,13 @@ exports.registeredUser = function(req, res, next) {
 }
 
 exports.isAuthenticated = function(req,res,next){
-  if (req.isAuthenticated())
-       return next();
+  console.log("inside auth")
```
Comment (hemangsk) [inline config/verify.js:43]: remove this

File: models/relation.js [hunk 4]
```diff
@@ -2,9 +2,7 @@
 var bcrypt = require('bcrypt-nodejs');
 module.exports = function(models){
   console.log("Relations are biuld successfully");
-  models.User.hasOne(models.Applicant);
-  models.vacancy.hasMany(models.candidates_2017);
+  models.User.hasOne(models.Applicant, { foreignKey: 'ApplicantId' , targetKey: 'UserId', foreignKeyContraint : true});
```
Comment (hemangsk) [inline models/relation.js:5]: remove foreignkeyconstraint

**Comment by hemangsk** [comment]
Comment on 72ded82cc99cd15a9363f543c4c86f2e6188ee59, file routes/index.js, line 135.

Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/routes/index.js
+++ b/routes/index.js
@@ -132,4 +132,4 @@
 // });
 
 
-module.exports = router;+module.exports = router;
```

**Comment by hemangsk** [comment]
@rultor merge

File: routes/dashboard.js [hunk 5]
```diff
@@ -21,12 +21,28 @@ router.post('/userinfo',verify.isAuthenticated, function(req, res, next) {
...
+              "FirstName" : req.body.first_name,
+              "LastName" : req.body.last_name,
+              "Discipline" : req.body.cse
+          }
+
           //Transaction started.
+    console.log(userinfo)
           models.sequelize.transaction(t=>{
             //Transaction Is used for only one session per user.
-            return models.applicant.create(userinfo).then(userinfo=>{
```
Comment (surender7790) [inline routes/dashboard.js]: What if an applicant if first time sending data to /userinfo route.  How can query find data which is not present in database.

Comment (surender7790) [inline routes/dashboard.js]: i am unable to submit data in applicant table using dashboard

File: models/candidates_2017.js (hunk 1 above)
Comment (hemangsk) [reply models/candidates_2017.js:5]: suppose some candidate want specialized fields for every application, like references for the job of associate prof and assistant prof in another subject may be different.

Comment (surender7790) [reply models/candidates_2017.js:5]: done. 👍

**Pull request merged**